share-extension/ShareViewController.m

```diff
@@ -82,7 +82,7 @@ @implementation ShareViewController
 #pragma mark - Share VC life cycle
 
 - (BOOL)isContentValid {
-    return self.contentText != nil;
+    return self.contentText != nil && loggingStatus >= eXoStatusLoggedIn && loggingStatus != eXoStatusLoadingSpaceId; // all status >= eXoStatusLoggedIn means user have logged in.
```

Comment:
This method is not used. Please check if it should be used or not. If not then remove it.